integration/service/src/main/java/com/cerner/beadledom/integration/service/ResteasyBootstrapModule.java

```diff
@@ -0,0 +1,54 @@
+package com.cerner.beadledom.integration.service;
+
+import com.cerner.beadledom.health.HealthModule;
+import com.cerner.beadledom.metadata.BuildInfo;
+import com.cerner.beadledom.metadata.ServiceMetadata;
+import com.cerner.beadledom.pagination.OffsetPaginationModule;
+import com.cerner.beadledom.pagination.models.OffsetPaginationConfiguration;
+import com.cerner.beadledom.resteasy.ResteasyModule;
+import com.cerner.beadledom.swagger2.Swagger2Module;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.multibindings.OptionalBinder;
+import io.swagger.models.Info;
+import javax.inject.Singleton;
+
+/**
+ * A Guice Module that installs the {@link ResteasyModule}, {@link Swagger2Module},
```

Comment:
nit: might be better to use List tag to list them

Reply:
Changed to use the List tag: https://github.com/cerner/beadledom/pull/149/commits/802c74c005968d527aef7a82390173d07569d0d5